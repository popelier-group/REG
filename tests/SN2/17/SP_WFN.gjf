# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.00744600   -0.00006000    0.00019300
 H                 -0.90186800   -0.90682000   -0.56488000
 H                 -0.90184600    0.94270000   -0.50251100
 H                 -0.90192900   -0.03607500    1.06801900
 Br                 1.62807100    0.00000700   -0.00001800
 Cl                -3.25056400    0.00001000   -0.00004500

frame.wfn
