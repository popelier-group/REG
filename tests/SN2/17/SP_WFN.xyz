
6

C       -0.001397   -0.037554    0.891451
H       -0.603916   -0.914488    0.747833
H       -0.463821    0.929134    0.825589
H        1.064155   -0.114019    0.784515
Br        0.003066    0.073256   -1.741732
Cl       -0.005608   -0.131721    3.132587
