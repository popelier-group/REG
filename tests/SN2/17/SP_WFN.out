 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.17/Gau-20201.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.17/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     20202.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.00745  -0.00006   0.00019 
 H                    -0.90187  -0.90682  -0.56488 
 H                    -0.90185   0.9427   -0.50251 
 H                    -0.90193  -0.03608   1.06802 
 Br                    1.62807   0.00001  -0.00002 
 Cl                   -3.25056   0.00001  -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.007446   -0.000060    0.000193
      2          1           0       -0.901868   -0.906820   -0.564880
      3          1           0       -0.901846    0.942700   -0.502511
      4          1           0       -0.901929   -0.036075    1.068019
      5         35           0        1.628071    0.000007   -0.000018
      6         17           0       -3.250564    0.000010   -0.000045
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.073624   0.000000
     3  H    1.073620   1.850571   0.000000
     4  H    1.073631   1.850556   1.850558   0.000000
     5  Br   2.635517   2.746269   2.746206   2.746435   0.000000
     6  Cl   2.243118   2.580262   2.580235   2.580339   4.878635
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.001397   -0.037554    0.891451
      2          1           0       -0.603916   -0.914488    0.747833
      3          1           0       -0.463821    0.929134    0.825589
      4          1           0        1.064155   -0.114019    0.784515
      5         35           0        0.003066    0.073256   -1.741732
      6         17           0       -0.005608   -0.131721    3.132587
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         146.4282441           0.8493292           0.8493291
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.1881941016 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  5.25D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.38926934     A.U. after   12 cycles
            NFock= 12  Conv=0.14D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.59484-101.34713 -62.24797 -56.06639 -56.06536
 Alpha  occ. eigenvalues --  -56.06536 -10.09107  -9.26015  -8.45684  -7.02205
 Alpha  occ. eigenvalues --   -7.01652  -7.01652  -6.28198  -6.27849  -6.27849
 Alpha  occ. eigenvalues --   -2.39395  -2.39299  -2.39299  -2.39024  -2.39024
 Alpha  occ. eigenvalues --   -0.62245  -0.55715  -0.51610  -0.30803  -0.30803
 Alpha  occ. eigenvalues --   -0.19007  -0.13196  -0.13196  -0.08706  -0.08642
 Alpha  occ. eigenvalues --   -0.08642
 Alpha virt. eigenvalues --    0.09776   0.11521   0.15205   0.15206   0.16391
 Alpha virt. eigenvalues --    0.17370   0.18929   0.18929   0.20679   0.21289
 Alpha virt. eigenvalues --    0.21289   0.22793   0.22793   0.24710   0.29608
 Alpha virt. eigenvalues --    0.31607   0.31608   0.37076   0.39184   0.54649
 Alpha virt. eigenvalues --    0.54649   0.55429   0.57890   0.58509   0.58509
 Alpha virt. eigenvalues --    0.62904   0.62905   0.69288   0.77276   0.77277
 Alpha virt. eigenvalues --    0.79244   0.82705   0.84749   0.84750   0.85051
 Alpha virt. eigenvalues --    0.85051   0.91801   1.05974   1.05975   1.06962
 Alpha virt. eigenvalues --    1.06962   1.10795   1.17913   1.23025   1.23026
 Alpha virt. eigenvalues --    1.29073   1.52858   1.62664   1.62666   1.71104
 Alpha virt. eigenvalues --    1.96799   1.98410   1.98411   2.13718   2.13718
 Alpha virt. eigenvalues --    2.21563   2.21563   2.31852   2.45349   2.45349
 Alpha virt. eigenvalues --    2.53832   2.61987   2.61987   2.68326   2.76689
 Alpha virt. eigenvalues --    2.76689   3.01749   3.20568   3.86157   3.86158
 Alpha virt. eigenvalues --    4.36121   4.36123   4.37181   4.37181   4.55255
 Alpha virt. eigenvalues --    6.76333   7.76866   7.76871   7.84348   9.85224
 Alpha virt. eigenvalues --   24.03765  26.01410  26.01411  26.29680  48.06967
 Alpha virt. eigenvalues --  215.83112 289.97533 289.97533 290.049761020.83574
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.168482   0.421213   0.421218   0.421200   0.004748  -0.059514
     2  H    0.421213   0.512299  -0.033727  -0.033739  -0.022984  -0.028399
     3  H    0.421218  -0.033727   0.512296  -0.033736  -0.022991  -0.028400
     4  H    0.421200  -0.033739  -0.033736   0.512307  -0.022964  -0.028395
     5  Br   0.004748  -0.022984  -0.022991  -0.022964  35.781613   0.022789
     6  Cl  -0.059514  -0.028399  -0.028400  -0.028395   0.022789  17.560367
 Mulliken charges:
               1
     1  C   -0.377347
     2  H    0.185338
     3  H    0.185341
     4  H    0.185328
     5  Br  -0.740211
     6  Cl  -0.438449
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.178660
     5  Br  -0.740211
     6  Cl  -0.438449
 Electronic spatial extent (au):  <R**2>=           1140.0041
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0022    Y=              0.0424    Z=             -1.0111  Tot=              1.0120
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.9816   YY=            -44.0510   ZZ=            -83.4056
   XY=             -0.0030   XZ=              0.0705   YZ=              1.6578
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             13.1645   YY=             13.0950   ZZ=            -26.2595
   XY=             -0.0030   XZ=              0.0705   YZ=              1.6578
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.1315  YYY=              1.9479  ZZZ=           -109.8447  XYY=             -1.0333
  XXY=              0.3211  XXZ=            -13.4692  XZZ=              0.1457  YZZ=              3.4874
  YYZ=            -13.5750  XYZ=             -0.0501
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -72.3679 YYYY=            -74.7721 ZZZZ=          -1670.2547 XXXY=             -0.0808
 XXXZ=              1.9377 YYYX=              0.0429 YYYZ=             28.5301 ZZZX=              1.6374
 ZZZY=             38.8565 XXYY=            -24.5010 XXZZ=           -248.7760 YYZZ=           -249.5905
 XXYZ=              9.2761 YYXZ=             -0.3372 ZZXY=             -0.0983
 N-N= 1.711881941016D+02 E-N=-7.695392800352D+03  KE= 3.069968080369D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE531\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.007446,-0.00006,0.000193\H,0,-0.901868,-0.90682,-0.56488\H,0,-0
 .901846,0.9427,-0.502511\H,0,-0.901929,-0.036075,1.068019\Br,0,1.62807
 1,0.000007,-0.000018\Cl,0,-3.250564,0.00001,-0.000045\\Version=ES64L-G
 16RevC.01\State=1-A\HF=-3074.3892693\RMSD=1.392e-09\Dipole=0.3981443,-
 0.0000554,0.0001801\Quadrupole=-19.5752302,9.7876681,9.7875622,0.00009
 98,-0.0004439,-0.0000134\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 KNIGHT- SIRE, I'VE JUST RETURNED FROM THE FRONT
 KING-   HOW GOES THE BATTLE                    
 KNIGHT- THE SITUATION IS QUITE FLUID           
 KING-   WHAT DOES *THAT* MEAN                  
 KNIGHT- WE'RE UP THE CREEK                     
                THE WIZARD OF ID                
 Job cpu time:       0 days  0 hours  0 minutes 28.4 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.6 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:49:16 2020.
